src/commands/nervous_system_parameters.rs

```diff
@@ -0,0 +1,18 @@
+use crate::lib::signing::{sign_query_as_ingress_with_request_id, IngressWithRequestId};
+use crate::lib::{AnyhowResult, TargetCanister};
+use crate::SnsCanisterIds;
+use candid::Encode;
+use ic_base_types::PrincipalId;
+
+pub fn exec(sns_canister_ids: &SnsCanisterIds) -> AnyhowResult<Vec<IngressWithRequestId>> {
+    let governance_canister_id = PrincipalId::from(sns_canister_ids.governance_canister_id).0;
+    let args = Encode!()?;
+
+    let msg = sign_query_as_ingress_with_request_id(
+        "",
```

Comment:
I think this is an indicator that this is the wrong signing method to use for this governance API (unless there is some non-intuitive behavior which means this probably deserves a comment). `get_nervous_system_parameters` is a query call so there's no need to sign the message (you left the pem blank already). There is also no need to submit a request id to send another message to retrieve the reply. I suggest you use the `send_unsigned_ingress` method, you can see an example here https://github.com/dfinity/sns-quill/blob/master/src/commands/account_balance.rs#L29-L35

Reply:
I structured this way because we want the summary command to batch the messages, and print them, so that they can be sent all together with the send command (like the stake neuron command). Its much easier if there are all IngressMessagesWithRequestId. An option may be to have this behaviour as an option, and if the command is called directly use a query as a deffault, but if called from the summary command, then use and update method and " sign_query_as_ingress_with_request_id".

wdyt?

Reply:
Ah understood. Then I think using `sign_query_as_ingress_with_request_id` is fine. Can you add a comment to above this line explaining this? I think anyone reading this without understanding that context will think its a bug

Reply:
done